src/util/narrow.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019 Diffblue Limited. All Rights Reserved.
```

Comment:
Please sync within Diffblue whether that's a copyright mark to use.

Reply:
Copyright updated.